src/test/scala/com/johnsnowlabs/nlp/annotators/ner/dl/NerDLSpec.scala

```diff
@@ -337,9 +337,13 @@ class NerDLSpec extends AnyFlatSpec {
     val awsSecretAccessKey = sys.env("AWS_SECRET_ACCESS_KEY")
     val awsSessionToken = sys.env("AWS_SESSION_TOKEN")
 
-    ResourceHelper.getSparkSessionWithS3(awsAccessKeyId, awsSecretAccessKey, awsSessionToken)
+    ResourceHelper.getSparkSessionWithS3(
+      awsAccessKeyId,
+      awsSecretAccessKey,
+      "3.3.1",
```

Comment:
Since `3.3.1` is a hardcoded configuration value, we could define it in `ConfigLoader` object.